### PR TITLE
Add size definition for Infineon_PG-DSO-8-27

### PR DESCRIPTION
**Pull request by poeschlr**
Footprint pull request: https://github.com/KiCad/kicad-footprints/pull/1633

### CODE REVIEW

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/infinion.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  overall_size_x: 6.0 +/-0.2
+  lead_width: 0.41 +/-0.09
+  lead_len: 0.64 +/-0.25
+
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 4
+
+  EP_size_x: 2.64 +/-0.2
```
Comment (evanshultz) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/infinion.yaml]: Should be 2.65

**Comment by evanshultz** [comment]
Where did you get `HSOIC` from? Is it called out for this package?

**Comment by poeschlr** [comment]
possibly a copy paste error (i duplicated the hsioc file to generate this)

**Comment by evanshultz** [comment]
Alright. If you make the appropriate fixes I can merge the footprint and approve this.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/infinion.yaml (hunk 1 above)
Comment (poeschlr) [reply scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/infinion.yaml]: done

**Comment by evanshultz** [comment]
A few things:
1. Typo of `Infineon` (`i` not `e`).
1. Should the manufacturer name (line 7) move to the top of the YAML file since the file is manufacturer-specific?
1. Since no footprint has been merged, shall we include body size and lead pitch in the footprint name to help users find it and make the footprint more conformant with our symbol footprint filters? I can understand it's fully-defined since it's manufacturer-specific, but having that extra info may help searchability since it's not in the description or keywords.
1. There is a double space in the description of the resulting footprint. Do you want to fix that before merging? I assume it's due to a missing parameter in the YAML file that the description generator expects to be present.

**Comment by chschlue** [comment]
`Infineon` is correct. `infinion.yaml` is wrong.

logo-desktop-en

**Comment by poeschlr** [comment]
Well one would think i could write the name of the company i work for. Don't tell my boss.

**Comment by poeschlr** [comment]
Regarding point 2: not supported by the script right now.
Added body size
The double space does not come from the yaml file so no this would be something that woudl need fixing somewhere else.

**Comment by evanshultz** [comment]
Merged the footprint. Double spaces can be fixed elsewhere then.

@pointhi 
This is ready to be merged.

**Pull request merged**